runtime/mainnet/src/genesis.rs

```diff
@@ -0,0 +1,610 @@
+use alloc::{vec, vec::Vec};
+
+use cumulus_primitives_core::ParaId;
+use parachains_common::{AccountId, AuraId, Balance};
+use pop_runtime_common::genesis::*;
+use sp_core::crypto::Ss58Codec;
+use sp_genesis_builder::PresetId;
+use sp_runtime::traits::AccountIdConversion;
+
+use crate::{
+	config::{
+		collation::PotId,
+		governance::SudoAddress,
+		monetary::{ExistentialDeposit, MaintenanceAccount, TreasuryAccount},
+	},
+	AssetsConfig, BalancesConfig, CouncilConfig, Runtime, SessionKeys, EXISTENTIAL_DEPOSIT, UNIT,
+};
+
+/// A development chain running on a single node, using the `mainnet` runtime.
+pub const MAINNET_DEV: &str = "pop-dev";
+/// Configures a local chain running on multiple nodes for testing purposes, using the `mainnet`
+/// runtime.
+pub const MAINNET_LOCAL: &str = "pop-local";
+/// A live chain running on multiple nodes, using the `mainnet` runtime.
+pub const MAINNET: &str = "pop";
+/// The available genesis config presets;
+const PRESETS: [&str; 3] = [MAINNET_DEV, MAINNET_LOCAL, MAINNET];
+
+/// The parachain identifier to set in genesis config.
+pub const PARA_ID: ParaId = ParaId::new(3395);
+
+/// Initial balance for genesis endowed accounts.
```

Comment:
Nit: because this is the mainnet config, we this might be misleading at first glance and make someone thing we are endowing accounts with balance at genesis.

Could be worthwhile to clarify this is genesis endowed accounts for local development only.
```suggestion
/// Initial balance for genesis endowed accounts for local testing only.
```

Reply:
[1ea9626](https://github.com/r0gue-io/pop-node/pull/473/commits/1ea962642dad3a32b4908d8f0a501df8fc0e527c)